bundles/org.eclipse.ui.ide/extensions/org/eclipse/ui/actions/WorkspaceModifyOperation.java

```diff
@@ -103,20 +103,17 @@ protected abstract void execute(IProgressMonitor monitor)
 	@Override
 	public synchronized final void run(IProgressMonitor monitor)
 			throws InvocationTargetException, InterruptedException {
-		final InvocationTargetException[] iteHolder = new InvocationTargetException[1];
+		final Exception[] exceptionHolder = new Exception[1];
```

Comment:
Can you please change this into `AtomicReference` or similiar? Misusing arrays for passing references is a bad pattern, both in terms of comprehensibility as well because it's not thread-safe (no problem here, but bad example for other places).

Reply:
Done in 12615e9efe647076b42e23ad1130bd965296e783